docs/perf/usage/index.md

```diff
@@ -86,6 +86,25 @@ async function customTrace() {
 }
 ```
 
+## Custom screen traces
+
+Record a custom screen rendering trace (slow frames / frozen frames)
+
+```jsx
+import perf from '@react-native-firebase/perf';
+
+async function screenTrace() {
+  // Define & start a screen trace
+  try {
+    const trace = await perf().startScrenTrace('FooScreen');
+    // Stop the trace
+    await trace.stop();
+  } catch (e) {
+    // rejects if iOS or < Android 9.0
```

Comment:
```suggestion
    // rejects if iOS or (Android == 8 || Android == 8.1)
```
It's actually just API26 and API27, Android 5, 6, 7 etc, totally fine.

Reply:
Question on this. ScreenTrace is currently used in the auto instrumentation. Would we be bifurcating `Android == 8 || Android == 8.1` support if we reject these custom screen traces on Android 8 but allow the automatic screen tracing on Android 8?

Reply:
Maybe? I'm not sure what the performance module is doing under the covers for Android 8/8.1 all I know is what I have personally had to solve, and the solution I came up with, on an app with 2 million users and active use of 15mins / day (that is: a lot of users, enough I'm pretty confident in our work there). 

It doesn't look like the underlying code takes a position

seems to note if it cannot do it https://github.com/firebase/firebase-android-sdk/blob/fa4adcf92b354898396cac614b4e2149bb29c797/firebase-perf/src/main/java/com/google/firebase/perf/application/FrameMetricsRecorder.java#L113

but does not seem to throw if you create a trace without hardware acceleration and you only find out on stop?

Below firebase-android-sdk is https://developer.android.com/reference/androidx/core/app/FrameMetricsAggregator#add(android.app.Activity) which does not seem to do anything about it.

Perhaps instead of forcing it on here, except for API26/27 we should just document that it requires hardware acceleration, and throw if it is not on, with a side note indicating that it is on in almost all cases but there are valid reasons to turn it off in some cases e.g. API26/27+WebView.

I like that better as the idea of toggling a new code pathway on in view hierarchy seems like a very unexpected side effect, to me, of a measurement tool

Reply:
Good call, I'll make those changes.

Reply:
changes made